src/transformStateWithClones.js

```diff
@@ -7,7 +7,37 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let copyState = { ...state };
```

Comment:
Consider renaming `copyState` to `currentState` or `stateClone` for better readability, as it represents the current state after each action.